tutorials/algorithms/02_vqe_advanced_options.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Advanced VQE Options\n",
     "\n",
-    "In the first algorithms tutorial, you learned how to set up a basic [VQE](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQE.html) algorithm. Now, you will see how to provide more advanced configuration parameters to explore the full range of capabilities of Qiskit's variational algorithms: [VQE](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQE.html), [QAOA](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.QAOA.html) and [VQD](https://qiskit.org/documentation/stubs/qiskit.algorithms.eigensolvers.VQD.html) among others. In particular, this tutorial will cover how to set up a `callback` to monitor convergence and the use of custom `initial point`s and `gradient`s."
+    "In the first algorithms tutorial, you learned how to set up a basic [VQE](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQE.html) algorithm. Now, you will see how to provide more advanced configuration parameters to explore the full range of capabilities of Qiskit's variational algorithms: [VQE](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQE.html), [QAOA](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.QAOA.html) and [PVQD](https://qiskit.org/documentation/stubs/qiskit.algorithms.PVQD.html#qiskit.algorithms.PVQD) among others. In particular, this tutorial will cover how to set up a `callback` to monitor convergence and the use of custom `initial point`s and `gradient`s."
```

Comment:
This was correct as it was before. It was intended to point to the `VQD` eigensolver as its similar to VQE but can produces a number of states (excited states) as well the ground state.

Reply:
It was written in the documentation that VQD will be deprecated later and I also found the alternative so I replaced it.
I will revert that change. 
Thanks

Reply:
Yes, for minimum(eigensolvers) there exist the now deprecated versions and new versions. If it has `(minimum)_eigensolvers` where there is no `_` in `eigensolvers` ie. old version is `(minumum_)eigen_solvers` then its the new one (If also its one of these types and points to the convenience import direct in qiskit.algorithms then its the old one - we had to keep it this way to avoid breaking existing code - at some point in the future, after the old ones are removed, we may be able to have these point at the new ones). Hopefully these tutorials are correct in that we just went through them recently to update them to primitives and the new algos - the above though may helpful as you look at others just in case!